tests/IPv4Test.php

```diff
@@ -159,4 +159,35 @@ public function testHumanReadable($shortForm, $longForm)
         $this->assertEquals($shortForm, $ip->humanReadable());
         $this->assertEquals($longForm, $ip->humanReadable(false));
     }
+
+    /**
+     * @return array
+     */
+    public function getLoopbackTestAddresses(): array
+    {
+        return array(
+            array('127.0.0.0', '0.0.0.0'),
+            array('127.0.0.1', '1.1.1.1'),
+            array('127.10.0.1', '10.8.8.8'),
+            array('127.10.99.1', '10.20.30.40'),
+            array('127.10.99.87', '99.100.100.1'),
+            array('127.255.255.255', '255.255.255.255'),
+            array('127.255.255.0', '119.15.96.43'),
+        );
+    }
+
+    /**
+     * @dataProvider getLoopbackTestAddresses
+     *
+     * @param string $validLoopback
+     * @param string $invalidLoopback
+     */
+    public function testIsLoopback(string $validLoopback, string $invalidLoopback)
```

Comment:
This is a bit strange, why did you couple valid loopbacks with invalid loopbacks? You have 2 better options: 1) write 2 data providers and 2 tests, or 2) write a data provider that provides the IP + the expected return from isLoopback (true or false)